2-Intelligent-Agents/index.html

```diff
@@ -139,7 +140,7 @@ <h3>Evaluating multiple agents</h3>
           <h3>Evaluating many data sets</h3>
 
           TODO: one agent on many different data sequences (avoid overfitting)
-
+        </div>
```

Comment:
This </div> seems to be too many. Can you check this? It looks like you have 6 <div> and 7 </div> in this file.

Reply:
@redblobgames fixed it